regression/tests/enterprise/ent_test_base.py

```diff
@@ -126,12 +126,12 @@ def access_course(self):
         self.ent_portal_home.open_courses_popup()
         course_titles = self.ent_portal_home.fetch_course_titles_list()
         self.assert_(
-            ENT_COURSE_TITLE in course_title
+            self.ENT_COURSE_TITLE in course_title
```

Comment:
I think you have made a mistake while resolving conflicts
 in the last PR you have changed these vars from Class vars to Global Vars, now you are again changing these back